README.md

```diff
@@ -0,0 +1,9 @@
+# Synthesis Fork of Colyseus-Unity-SDK
+
+**When updating this fork, in order to make your changes bubble up to games consuming this package, you must do the following in each:**
+
+1. In the `manifest.json` file, remove the line for this package `"com.gosynthschool.io.colyseus.sdk": "https://github.com/gosynthschool/colyseus-unity-sdk.git#synthesis-changes",`.
```

Comment:
**Suggestion**: I'd emphasize just a bit more, however you like, that these instruction are for the game repos, not this repo.

```suggestion
1. In the `manifest.json` file of each game, remove the line for this package `"com.gosynthschool.io.colyseus.sdk": "https://github.com/gosynthschool/colyseus-unity-sdk.git#synthesis-changes",`.
```